scripts/main.js

```diff
@@ -0,0 +1,19 @@
+document.addEventListener('DOMContentLoaded', () => {
```

Comment:
[OPTIONAL] It would be nice and readable if you use separation of concern here by giving the eventlistener to each of the tags it wants to listen to rather than the whole document. This is more work for the document and reduces its efficiency.